hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/Reconciler.java

```diff
@@ -117,4 +124,42 @@ private void explicitlyReconcileTasks(SchedulerDriver driver) {
 
     driver.reconcileTasks(tasks);
   }
+
+  private class ReconcileThread extends Thread {
+    private static final int MILLIS_FROM_SECONDS = 1000;
```

Comment:
"It's just that we tend to be extremely picky and as careful as possible with the project to avoid any issues."

I wonder when this pickiness started?  As you know there was already a constant of the same style introduced here in HdfsScheduler.  The inconsistency between config keys was present before my change, but this PR is the one which is required to fix the problem.  In fact both of these problems were introduced on July 1.

There appears to be a shifting set of rules based on no hard and fast style guidelines, with inconsistent enforcement.  Having to address these incoherent style rule violations in every PR is tedious...